api/proto/config/devconfig.proto

```diff
@@ -153,3 +165,21 @@ message ConfigResponse {
   EdgeDevConfig config = 1;
   string configHash = 2;
 }
+
+// BootstrapConfig is used to carry initial device configuration, baked into a "single-use"
+// EVE installer prepared for a specific device to bootstrap, i.e. to establish the initial
+// connectivity with the controller and onboard. This is useful if the default network
+// configuration (i.e. Ethernet connectivity with DHCP) is not applicable for access
+// to the controller.
+message BootstrapConfig {
+  // Device configuration to use for bootstrapping.
+  // Should be signed by the controller using AuthContainer (see OBJECT_SIGNING.md).
+  // Put **protobuf-encoded** (not json) EdgeDevConfig under AuthContainer.protectedPayload.
+  // Instead of AuthContainer.senderCert, use controller_certs attribute below to include
+  // the signing certificate and all intermediate certificates needed for signature
+  // verification against root_certificate.
+  org.lfedge.eve.auth.AuthContainer signed_config = 1;
+  // Signing certificate (CERT_TYPE_CONTROLLER_SIGNING) and intermediate certificates
+  // (CERT_TYPE_CONTROLLER_INTERMEDIATE).
+  repeated org.lfedge.eve.certs.ZCert controller_certs = 2;
```

Comment:
Why not reuse message ZControllerCert? Then we can presumably reuse the function which processes those.

Reply:
I'm already reusing the same structure for certificate (ZCert).
It does not seem to be useful to have certs nested inside another structure:
```
message ZControllerCert {
  repeated ZCert certs = 1;  //list of certificates sent by controller
}
```

Reply:
The useful part is that you can call the existing VerifySigningCertChain function if you use message ZControllerCert

Reply:
IMHO `VerifySigningCertChain` is not very well defined by taking a vague argument `contents []byte`. The protobuf unmarshaling is not really related to signature verification. I will change it to take a list of controller certificates instead (`[]*ZCert`), so that it is clear what it needs.
Also, designing APIs based on (always-changing) implementation details seems strange.

Reply:
I will not be able to use the function directly anyway - it takes `contents []byte` as an argument and performs proto unmarshaling to ZControllerCert. However, in my case the outermost structure is BootstrapConfig.
I will split out the reusable part the function and define it to receive a list of ZCert instances as an argument.

Reply:
OK